### ISSUE TITLE
upper body person finder

### ISSUE THREAD

**Issue by OMARI1988** [shihenw/convolutional-pose-machines-release#34]
Hi,

I am trying to use the convolutional-pose-machines to find the pose of people in an office environment, where people are usually seated. My question is; is there a caffemodel that finds the upper body parts of a person without their lower body (when people are sitting the legs are usually occluded)? The first part of the code (person_net) is failing to find seated people.

Help is much appreciated

**Comment by shihenw** [comment]
You're right, the person detector here is not very strong. You can replace it to other person detector and use to result as the initialization of CPMs.
